packages/studio-ui/src/web/components/Layout/Sidebar.tsx

```diff
@@ -44,13 +44,6 @@ const BASIC_MENU_ITEMS = [
     rule: { res: 'module.qna', op: 'write' },
     icon: 'chat'
   },
-  {
-    id: 'libraries',
-    name: lang.tr('libraries.fullName'),
-    path: '/libraries',
-    rule: { res: 'module.code-editor', op: 'read' },
-    icon: 'book'
-  },
```

Comment:
So we disable it for both cloud and non cloud bots?

Reply:
yes, for this particular release. It is still accessible through url though